distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -479,6 +479,10 @@ private static Integer findPortNumber(Integer portToTry) {
 
   protected Option[] configurePaxExam() {
     return options(
+        when(Boolean.getBoolean("acdebuggerEnabled") && !Boolean.getBoolean("isDebugEnabled"))
```

Comment:
✏️ Can we move https://github.com/codice/ddf/pull/3867/files#diff-ad1ca21c3bbf51c2b25ccd0baefc7bb7R568 up here or move this block down there? It helps having our debug related configurations together.